development/process.md

```diff
@@ -104,6 +88,62 @@ keep the amount of rework from the contributor to a minimum.
       website are updated.
     * Finally the release is announced.
 
+
+## Maintainer processes
+
+[Maintainers](./roles.html#maintainer) support the contributions from the
+community and other maintainers, by participating in the contribution process.
+The following additional processes and steps apply for their work.
+
+### Pull request review
+
+The following aspects apply for reviewing pull requests.
+
+* Code and tests should be part of the pull requests and review.
+* Commit content, sequencing, and messages must be reviewed.
+* Documentation should be part of the PR or available in a separate PR, that is
+  ideally merged and managed together with the code contribution.
+* Maintainers can schedule additional workflow runs with tests that use external
+  services when reviewing a PR by adding a comment starting with this line:
```

Comment:
```suggestion
  services when reviewing a PR by adding a comment:
```

(this must be the sole entry in the comment though it might work even if there's trailing content but why risk false positives)

Reply:
Also maybe something about the security implications must be said here - that maintainers must check that the PR is not malicious or exfiltrating secrets/access tokens etc.